cli/src/main/java/com/devonfw/tools/ide/context/GitContextImpl.java

```diff
@@ -179,7 +179,7 @@ public void pull(Path targetRepository) {
     this.processContext.directory(targetRepository);
     ProcessResult result;
     // pull from remote
-    result = this.processContext.addArg("--no-pager").addArg("pull").run(PROCESS_MODE);
+    result = this.processContext.addArg("--no-pager").addArg("pull").addArg("--quiet").run(PROCESS_MODE);
```

Comment:
this will make every "pull" quiet.
However, a nice and pragmatic KISS solution.
If we later want to have situations where we do not want pull to be quiet, we can still add a boolean flag to this method. So I will already resolve this and merge the PR.